packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -179,6 +179,23 @@ class PlatformViewsService {
     return controller;
   }
 
+  /// Whether the render surface of the Android FlutterView should be converted.
+  ///
+  /// When adding platform views using
+  /// [Hybrid Composition](https://flutter.dev/docs/development/platform-integration/platform-views),
+  /// the engine converts the render surface to help improve animation
+  /// synchronization on Android. On Android versions < 10, this can have some
+  /// performance consequences. This flag allows disabling this conversion.
+  ///
+  /// Defaults to false.
+  static Future<void> disableConvertingRenderSurfaceOnAndroid(bool disabled) {
```

Comment:
it seems strange to have a `disable**` method name that takes a `boolean` as an argument.  What about `synchronizeToNativeViewHierarchy(bool yes)` ?